sdk/attestation/azure-security-attestation/samples/sample_authentication_async.py

```diff
@@ -99,27 +103,25 @@ async def create_attestation_client_shared(self):
 
         write_banner("create_attestation_client_shared")
         # [START sharedclient_create]
-        tenant_id = os.getenv("ATTESTATION_TENANT_ID")
-        client_id = os.getenv("ATTESTATION_CLIENT_ID")
-        secret = os.getenv("ATTESTATION_CLIENT_SECRET")
-
-        if not tenant_id or not client_id or not secret:
-            raise Exception("Must provide client credentials.")
-
-        # Create azure-identity class
-        from azure.identity.aio import ClientSecretCredential
 
+        # Import default credential and Attestation client
+        from azure.identity.aio import DefaultAzureCredential
         from azure.security.attestation.aio import AttestationClient
 
         shared_short_name = os.getenv("ATTESTATION_LOCATION_SHORT_NAME")
-        shared_url = 'https://shared' + shared_short_name + \
-            '.' + shared_short_name + '.attest.azure.net'
-
-        async with ClientSecretCredential(
-            tenant_id=tenant_id, client_id=client_id, client_secret=secret) as credentials, AttestationClient(
-                credentials, instance_url=self.aad_url) as client:
+        shared_url = (
+            "https://shared"
+            + shared_short_name
+            + "."
+            + shared_short_name
+            + ".attest.azure.net"
+        )
```

Comment:
Here's a one-liner you might want to use for these snippets in each sample file:
```suggestion
        shared_url = ("https://shared{}.{}.attest.azure.net".format(shared_short_name, shared_short_name))
```